bot.py

```diff
@@ -10,8 +10,9 @@
 from get_function.translate import get_your_translate
 
 bot = telebot.TeleBot(os.getenv('BOT_TOKEN'))
+logger.debug(f'The WeatherBot started')
```

Comment:
better to use logger that implemented in pyTelegramBotApi, but it's ok to use different library